src/runners/cairoRunner.test.ts

```diff
@@ -30,11 +31,26 @@ describe('cairoRunner', () => {
     test('should return the value of the 10th fibonacci number', () => {
       const runner = new CairoRunner(PROGRAM);
       const finalPc = new Relocatable(0, 12);
-      runner.runUntilPc(finalPc, false);
+      runner.runUntilPc(finalPc, false, true, true);
       const executionSize = runner.vm.memory.getSegmentSize(1);
       const executionEnd = runner.executionBase.add(executionSize);
 
       expect(runner.vm.memory.get(executionEnd.sub(1))).toEqual(new Felt(144n));
     });
+
+    test('should export encoded trace and memory', () => {
+      const runner = new CairoRunner(PROGRAM);
+      const finalPc = new Relocatable(0, 12);
```

Comment:
I see that you copy/pasted this from the test above, but you really don't want to do this because it's program dependant. What if your test program gets longer and 12 is not the first empty slot anymore?

In the python vm, it's just to random relocatable, got by creating new segments

Reply:
Yep, made a dummy copy/paste to quickly check that the encoding was working
Declaring a new variable `finalPc` instead of using the one initialized in the runner, `runner.finalPc` doesn't make sense anyway

Some more refactoring might be done in a later PR